common/templates/xblock_v2/xblock_iframe.html

```diff
@@ -327,19 +324,12 @@
                                 </ul>
                             </div>
                         `;
-                        element.innerHTML += xblockActions;
-
-                        const views = editorView.getMetadataEditor().views;
-                        Object.values(views).forEach(view => {
-                            const uniqueId = view.uniqueId;
-                            const input = element.querySelector(`#${uniqueId}`);
-                            if (input) {
-                                input.addEventListener("input", function(event) {
-                                    view.model.setValue(event.target.value);
-                                });
-                            }
-                        });
```

Comment:
@ChrisChV What was the reason for this piece of code? It is causing issues with fields that want to update model value on change instead of `input`, in this case the fields that are created by `Add` button loose focus each time you type something. Removing this listener fixed it and the editor works still works without any issues.

Reply:
The JavaScript functions that saved data to the model weren't working, so this code was added here. The model is used when saving. With your changes, these JavaScript functions now work. It's okay to delete this code. Thanks!